mmv1/third_party/terraform/resources/resource_bigtable_instance.go

```diff
@@ -376,14 +410,25 @@ func expandBigtableClusters(clusters []interface{}, instanceID string, config *C
 		case "HDD":
 			storageType = bigtable.HDD
 		}
-		results = append(results, bigtable.ClusterConfig{
+
+		cluster_config := bigtable.ClusterConfig{
 			InstanceID:  instanceID,
 			Zone:        zone,
 			ClusterID:   cluster["cluster_id"].(string),
 			NumNodes:    int32(cluster["num_nodes"].(int)),
 			StorageType: storageType,
 			KMSKeyName:  cluster["kms_key_name"].(string),
-		})
+		}
+		autoscaling_configs := cluster["autoscaling_config"].([]interface{})
+		if len(autoscaling_configs) > 0 {
```

Comment:
What happens if len(autoscaling_configs) > 1? Are we silently dropping the rest of the configs?